docs/datamodel/scalars/enum.rst

```diff
@@ -0,0 +1,27 @@
+.. _ref_datamodel_scalars_enum:
+
+Enumerated Types
+================
+
+.. eql:type:: std::enum
+
+    :index: enum
+
+    An enumerated type is a data type consisting of an order list of values.
+
+    An enumeration type can be declared in a schema declaration using
+    the following syntax:
+
+    .. eql:synopsis::
+
+        enum "<" <enum-values> ">"
```

Comment:
```scalar type <name> extending enum "<" <enum-values> ">"```

Reply:
This bit is specifically about the syntax for an anonymous enum type declaration.